code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -603,13 +656,6 @@
 
 		//This is for attacking.
 		if(target)
-			if(targets_from && isturf(targets_from.loc) && in_range)
-				//If they're next to us, attack
-				MeleeAction()
-			else
-				if(rapid_melee > 1 && target_distance <= melee_queue_distance)
-					MeleeAction(FALSE)
-				in_melee = FALSE //If we're just preparing to strike do not enter sidestep mode
```

Comment:
I'm ASSUMING this works by virtue of "Attacking is done automatically" correct?

Reply:
its moved into handle automated action instead of the move code as well as a response to bump and on move